pkg/agent/manager/sync.go

```diff
@@ -76,11 +82,9 @@ func (m *manager) processTaintedAuthorities(ctx context.Context, bundle *spiffeb
 		}
 	}
 
-	newTaintedJWTAuthorities := getNewItems(m.processedTaintedJWTAuthorities, jwtAuthorities)
-	if len(newTaintedJWTAuthorities) > 0 {
-		m.c.Log.WithField(telemetry.SubjectKeyIDs, strings.Join(newTaintedJWTAuthorities, ",")).
-			Debug("New tainted JWT authorities found")
-		// TODO: IMPLEMENT!!!
+	if len(jwtAuthorities) > 0 {
```

Comment:
may we compare new tainted authorities with already tainted authorities? 
if not we'll be creating logs and metrics for something that there must be empty, right?